README.rst

```diff
@@ -284,6 +284,9 @@ extended in git-config(1) files::
   [gitpublishprofile "default"]
   suppresscc = all            # do not auto-cc people
 
+Additionally, if the default profile is empty and no --profile command-line
+option was given, git-publish will use the first profile it finds.
+
```

Comment:
The commit description says ".gitpublish tends to not specify the 'default' profile so it does not
interfere with user's defaults" but the commit seems to purposely defeat this.  Why not adjust .gitpublish to use "default" as the profile name if it should be used by default?